ansible/roles/debops.nullmailer/defaults/main.yml

```diff
@@ -234,6 +242,10 @@ nullmailer__configuration_files:
     content: '{{ nullmailer__sendtimeout }}'
     state: '{{ "present" if nullmailer__sendtimeout else "absent" }}'
 
+  - dest: '/etc/nullmailer/allmailfrom'
+    content: '{{ nullmailer__allmailfrom}}'
```

Comment:
You should put a space on both sides of the variable in braces. This will probably be caught by `ansible-lint` test.